cmd/optimizer-nri-plugin/main.go

```diff
@@ -86,6 +87,12 @@ func buildFlags(args *PluginArgs) []cli.Flag {
 			Usage:       "the directory to persist accessed files list for container",
 			Destination: &args.Config.PersistDir,
 		},
+		&cli.BoolFlag{
+			Name:        "readable",
```

Comment:
Is CSV the only output format now?

Reply:
In addition to the line-by-line file path list, Yes.